### ISSUE TITLE
KB translations don't inherit any of the topics of the parent article

### ISSUE THREAD

**Issue by akatsoulas** [mozilla/sumo#1312]
It turns out that a KB article translated to any locale, doesn't inherit the topics of the parent article. Although the discovery of the fallback and the translated documents leverages the parent/child relationship of the article, it should inherit the topics. We should also investigate if this holds true for the products too.

**Comment by smithellis** [comment]
@akatsoulas - I'm looking for some more information on this; I haven't (so far) been able to replicate the issue locally. If I create an article and then create a localization, the localized doc does inherit both Product and Topics. I'm wondering if the fail case is more specific?

**Comment by smithellis** [comment]
Ok, I recreated the problem.

**Comment by emilghittasv** [comment]
I have recreated this issue after building SUMO locally without having this fix in.

I can confirm that translation are successfully, on creation, inheriting parent article's topics and this can be seen inside the translated article breadcrumbs & inside the highlighted topic options from "Topics" section of the admin panel.

During the verification of this ticket I've also noticed that the "Product" is not being highlighted for the translated article inside the admin panel -> https://github.com/mozilla/sumo/issues/1487 and the topics for the translated articles are not automatically updated if the topics for the parent article have been changed -> https://github.com/mozilla/sumo/issues/1488

I'm moving this ticket inside the "Release" column.
